xml/vk.xml

```diff
@@ -143,23 +143,23 @@ server.
         <type requires="ggp_c/vulkan_types.h" name="GgpFrameToken"/>
 
         <type category="define">#define <name>VK_MAKE_VERSION</name>(major, minor, patch) \
-    (((major) &lt;&lt; 22) | ((minor) &lt;&lt; 12) | (patch))</type>
-        <type category="define">#define <name>VK_VERSION_MAJOR</name>(version) ((uint32_t)(version) &gt;&gt; 22)</type>
-        <type category="define">#define <name>VK_VERSION_MINOR</name>(version) (((uint32_t)(version) &gt;&gt; 12) &amp; 0x3ff)</type>
+    (((major) &lt;&lt; 22U) | ((minor) &lt;&lt; 12U) | (patch))</type>
```

Comment:
I don't see why the second operand should be unsigned. Positive literal number is perfectly normal and good code. This should probably not be needed.

Reply:
Nice spot, I missed there was an option to relax the warning constraint.
Thank you!